Makefile

```diff
@@ -147,6 +147,18 @@ endif
 	CC=$(CC) go install -a -v -tags "$(TAG_SQLITE3)" $(DEBUG) ./...
 	@echo "LXD built successfully"
 
+race:
```

Comment:
@stgraber the `debug` and `nocache` targets don't have the `CGO_ENABLED=0` steps for `lxd-p2c` and `lxd-agent`, shall I add them?

Reply:
Ah yeah, we should do that